dev-haskell/cryptonite/cryptonite-0.30.ebuild

```
@@ -22,11 +22,11 @@ RDEPEND=">=dev-lang/ghc-8.4.3:=
 "
 DEPEND="${RDEPEND}
 	>=dev-haskell/cabal-2.2.0.1
-	test? ( dev-haskell/memory
-		dev-haskell/tasty
-		dev-haskell/tasty-hunit
-		dev-haskell/tasty-kat
-		dev-haskell/tasty-quickcheck )
+	>=dev-haskell/memory-0.14.18
```

This is a runtime dependency and it should have:

- `=` as the slot which tells portage to rebuild `cryptonite` if the subslot of `memory` changes:

  > =
  > Indicates that any slot value is acceptable. In addition, for runtime dependencies, indicates that the package will break unless a matching package with slot and sub-slot equal to the slot and sub-slot of the best version installed as a build-time (DEPEND) dependency is available. 

  See: https://projects.gentoo.org/pms/8/pms.html#x1-820008.3.3

- `[profile?]` which indicates the `profile` USE flag on `cryptonite` must be enabled if it is enabled on `memory` (this is for ["profiling" in haskell](https://cabal.readthedocs.io/en/3.4/cabal-project.html#cfg-field-profiling)):

  >  [opt?]
  >    The flag must be enabled if the flag is enabled for the package with the dependency. 

  See: https://projects.gentoo.org/pms/8/pms.html#x1-820008.3.4

In other words, it should be in `RDEPEND` and look like:

```suggestion
	>=dev-haskell/memory-0.14.18:=[profile?]
```

---

Thank you, I've fixed it. As always your comments are really helpful and valuable.